class-plugin-autoupdate-filter.php

```diff
@@ -14,29 +14,104 @@ class Plugin_Autoupdate_Filter {
 	/**
 	 * Initialize WordPress hooks
 	 */
-	public function init() {
+	public function init(): void {
+
+		// get the centralized settings from opsoasis
+		try {
+			$this->settings = $this->get_auto_update_settings();
+		} catch ( Exception $exception ) {
+
+			$error_message  = $exception->getMessage();
+			$this->settings = (object) array( 'disable_all' => true );
+
+			error_log( "Plugin Autoupdate Filter: Unable to retrieve the autoupdate settings ({$error_message})" ); // phpcs:disable WordPress.PHP.DevelopmentFunctions
+			add_action(
+				'admin_notices',
+				function() use ( $error_message ) {
+					echo '<div class="notice notice-error"><p><strong> Plugin Autoupdate Filter:</strong> Unable to get autoupdate settings (' . esc_html( $error_message ) . ').</p></div>';
+				}
+			);
+		}
 
 		// setup plugins and core to autoupdate _unless_ it's during specific day/time
 		add_filter( 'auto_update_plugin', array( $this, 'auto_update_specific_times' ), 10, 2 );
 		add_filter( 'auto_update_core', array( $this, 'auto_update_specific_times' ), 10, 2 );
 
 		// Replace automatic update wording on plugin management page in admin
-		add_filter( 'plugin_auto_update_setting_html', array( $this, 'plugin_autoupdate_filter_custom_setting_html' ), 11, 3 );
+		add_filter( 'plugin_auto_update_setting_html', array( $this, 'custom_setting_html' ), 11, 3 );
```

Comment:
Since we're renaming methods ... I already like the new names, but wanted to bring up my naming scheme since it seems the appropriate time. When it comes to naming filters, I usually name it something like `filter_<value>` because the name of the method will then tell me that this is a) a filter not an action, b) what/why are we filtering.

This particular one, for example, I would name `filter_plugin_auto_update_settings_html`. The ones below I'd name `filter_auto_update_emails` and then `filter_auto_updates_debug_email`. 

The next action on `admin_init` would be something like `output_auto_updates_disabled_notice`. When looking at the function name, I'd immediately know that a) it's an action, not a filter, b) it outputs something to the screen, c) that something is most likely an admin notice;

Just a suggestion that I thought was timely, no need to action on it. Love the current changes already!

Reply:
Suggestions taken!